docs/simplesamlphp-customauth.md

```diff
@@ -272,6 +273,10 @@ The class follows:
                 throw new Exception('Missing or invalid password option in config.');
             }
             $this->password = $config['password'];
+            $this->options = $config['options'];
+            if (!is_array($config['options])) {
+                throw new Exception('Missing or invalid options option in config.');
+            }
```

Comment:
In this example `options` is now mandatory in the configuration while in the code below it remains optional, is that intentional?